manifests/data/defaults.yaml

```diff
@@ -0,0 +1 @@
+curator::package_name: 'elasticsearch-curator'
```

Comment:
is this file mandatory ? 🤔

Reply:
Apologies this is accidental , this should of been in the root of the module. I was looking at prepping the module for puppet 5 with native hiera support for default variables, however it appears I placed defaults.yaml in the wrong path.  https://puppet.com/docs/puppet/5.0/hiera_migrate_modules.html

Reply:
ah brilliant thanks 👍looks better with thie `hiera.yaml` :)

Reply:
I moved to these to separate branch on for now.